src/ol/source/tileimagesource.js

```diff
@@ -163,6 +163,36 @@ ol.source.TileImage.prototype.getTileCacheForProjection = function(projection) {
 
 
 /**
+ * @param {number} z Tile coordinate z.
+ * @param {number} x Tile coordinate x.
+ * @param {number} y Tile coordinate y.
+ * @param {number} pixelRatio Pixel ratio.
+ * @param {ol.proj.Projection} projection Projection.
+ * @param {string} key Key.
```

Comment:
I think this parameter needs a bit more explaining, also why it needs to be set on the tile instance.